src/js/components/Alert.js

```diff
@@ -0,0 +1,58 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import Icon from './Icon';
+
+const Alert = ({children, showIcon, type}) => {
+  const classes = classNames('alert', {
+    [`alert-${type}`]: type != null
+  });
+  let icon = null;
+
+  if (showIcon) {
+    const className = 'alert-icon';
+    const family = 'mini';
+    const size = 'mini';
+
+    switch (type) {
```

Comment:
What do we think about this pattern?
```js
const id = {
  danger: 'ring-exclamation', 
  success: 'checkmark'
}[type];
```

We could also go with a private component:
```js
const AlertIcon = ({type}) => {
  const id = {
    danger: 'ring-exclamation', 
    success: 'checkmark'
  }[type];

  return <Icon className="alert-icon" family="mini" id={id} size="mini" />;
};